src/main/target/common_post.h

```diff
@@ -552,14 +552,31 @@
 #undef USED_TIMERS
 #endif
 
-#if !defined(USE_RANGEFINDER)
+#if defined(USE_RANGEFINDER_HCSR04) || defined(USE_RANGEFINDER_SRF10) || defined(USE_RANGEFINDER_HCSR04_I2C) || defined(USE_RANGEFINDER_VL53L0X) || defined(USE_RANGEFINDER_UIB) || defined(USE_RANGEFINDER_TF) || defined(USE_RANGEFINDER_MT)
+#ifndef USE_RANGEFINDER
+#define USE_RANGEFINDER
+#endif
+#endif // USE_RANGEFINDER_XXX
+
+#ifndef USE_RANGEFINDER
 #undef USE_RANGEFINDER_HCSR04
 #undef USE_RANGEFINDER_SRF10
 #undef USE_RANGEFINDER_HCSR04_I2C
 #undef USE_RANGEFINDER_VL53L0X
 #undef USE_RANGEFINDER_UIB
 #undef USE_RANGEFINDER_TF
+#undef USE_RANGEFINDER_MT
+#endif // USE_RANGEFINDER
```

Comment:
This doesn't make sense. If any of the rangefinders is defined, this will never be true. And if none of the rangefinders are defined, this is not needed.

Reply:
This is old macro code - only changed to `ifndef` and added rangefinder_mt - but might be removed if you insist :joy:

Reply:
This PR intents to ease user inclusion of this hardware by only supplying the hardware device so the feature is added automatically.

Reply:
I think the point is if any are defined, the second undef block will never fire. Which implies there'd be no need for the undef block.